app/controllers/dashboard_controller.rb

```diff
@@ -7,6 +8,21 @@ def index
     @activities = Activity.all(current_user.token)
   end
 
+  def save_metrics_to_database
+    recent = @activities.select do |activity|
+      activity.recent == true
+    end
+
+    current_user.time_ridden = recent.reduce(0) do |sum, activity|
+      sum += activity.raw_time
+    end
+
+    current_user.distance_ridden = recent.reduce(0) do |sum, activity|
+      sum+= activity.distance
```

Comment:
Useless assignment to variable - `sum`. Use just operator `+`.<br>Surrounding space missing for operator '+='.